CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 
 ## [1.0.0] - 2020-08-31
 
+- Implement `getAllowedResourceTypesForOperation`

```

Comment:
```suggestion
## [1.1.0] - 2020-09-25

- feat: Implement `getAllowedResourceTypesForOperation`. This will allow the router to filter results before returning to the caller

```

Reply:
I think it's weird to mention the router implementation details here since router is a different package. I did fix the version/date though.